docs/source/quickstart.rst

```diff
@@ -1,81 +1,187 @@
+
+.. image:: ../media/stage3_quickstart.png
+
+Stage 4: Create Lightcurves
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Stage 4 takes the 1D spectra produced by the previous stage and turns them in to light curves. The number of wavelength channels to turn into light curves, along with the wavelength range across which they will be calculated, can be defined in the Stage 4 ``.ecf`` file. In the interest of reducing the computational burden of the following light curve fitting stage, only two light curves will be generated corresponding to 1.5-3.0 μm and 3.0-4.5 μm (see figure below). 
+
+Similarly to Stage 3, the actual data for the produced light curves can be found in the ``*Table_Save.txt`` file.
+
+.. image:: ../media/stage4_quickstart.png
+
+Stage 5: Lightcurve Fitting
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Stage 5 takes all of the lightcurves produced by the previous stage and performs a variety of fitting routines to estimate specific system and planetary properties. For this quickstart, the fitting was performed using nested sampling
+as implemented by ``dynesty``, for a model assuming a transit of WASP-39b plus an aribitrary linear polynomial trend. 
+
+As a reminder, the input initial guesses and priors for the model properties are contained within the Stage 5 ``.epf`` file. To facilitate this quickstart demo, input parameters applicable to WASP-39b have already been assigned. For your own reductions, you'll need to tailor this file to the system you are observing and the type of fit you want to perform. 
+
+Additionally, nested sampling is not the only method of sampling the posterior distributions, and MCMC as implemented by ``emcee`` can also be used. Gaussian processes as implemented by ``george`` can also be included in the overall fitting routine. 
+
+An example figure demonstrating the best fit model lightcurve alongside the data is shown below, and corner plot representations of the fit posteriors can be found under the ``figs`` directory. Once again, the actual model light curve data can be found in the ``*Table_Save_ch*.txt`` files. 
+
+.. image:: ../media/stage5_quickstart.png
 
-4. Run Eureka!
------------------------------------------------------------------
+Stage 6: Plot Spectra
+~~~~~~~~~~~~~~~~~~~~~
 
-4.1.  Now you're ready to run ``Eureka!``
+The final Stage of ``Eureka!``, Stage 6, takes the output data from the lightcurve fitting and produces transmission and/or emission spectra. As mentioned earlier, this quickstart only makes use of two different light curves from this dataset from 1.5-3.0 μm and 3.0-4.5 μm. In this case, our transmission spectrum for the transit of WASP-39b will only have two data points (see figure below). Note that the errors bars are not representative of what could be expected for true JWST data, as this dataset has been trimmed down from 8192 integrations, to only 32. Finally, the transmission spectrum data is saved in the ``*Table_Save.txt`` file. 
```

Comment:
No comma before "to only 32"

Reply:
I'd also explain in that sentence why people are only using 32 integrations instead of 8192